relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -8,6 +8,8 @@ use sqlparser::ast::{
 };
 use sqlparser::dialect::{Dialect, GenericDialect};
 
+const MAX_DEPTH: usize = 64;
```

Comment:
Max depth of what is it? Nested queries (sub-queris)? Maybe `MAX_NESTED_QUERIES` sounds better here?
Or each element of the statement we currently parse (e.g. Declre, Fetch, Savepoint etc?). 
It would be great to  some docs here, or/and change the name to better reflect the use case.

Reply:
It's supposed to limit the level of nesting within a query. I simplified the way it's measured now, + added some docs.